### ISSUE TITLE
Feature: Icinga for windows version

### ISSUE THREAD

**Issue by MarcusCaepio** [Icinga/icinga-powershell-plugins#289]
Hi all,
it would be nice to have a check, which alerts me, when a new version of Icinga for windows is available. Basically sth. like the apt check for debian/ubuntu. Could this be implemented?

Cheers,
Marcus

**Comment by tectumopticum** [comment]
You might check https://packages.icinga.com/IcingaForWindows/stable/ifw.repo.json or your local synchronized repo in regular intervals via check_http or Invoke-IcingaCheckHTTPStatus?

**Comment by MarcusCaepio** [comment]
Basically, I sth. like the icinga check for linux, which tells me since when icinga is running and optionally a "icinga_min_version" so I get an alert, when an agent is out of date. Just to know that there is a new version is not enough. I would like to know, where the new version is not installed yet :)

**Comment by tectumopticum** [comment]
Hm, the "icinga"-check is also working for windows-agents. I assume, you're talking about the output "Icinga 2 has been running for 1 hour, 36 minutes and 10 seconds. Version: v2.12.3", correct?

**Comment by MarcusCaepio** [comment]
Yes. I could not find a command for the icinga check agent in the generated commands.conf from the agent

**Comment by tectumopticum** [comment]
The command is listed in the "External Commands" (in the Director). Preview:
```
object CheckCommand "icinga" {
    import "plugin-check-command"
    timeout = 1m
    vars.icinga_min_version = ""
}
```

**Comment by tectumopticum** [comment]
Just use the notification-features of github. E.g. subscribe by using the "Watch"-button and customize it for your needs. You'll receive an e-mail about all new releases etc.
